exporter/elasticsearchexporter/README.md

```diff
@@ -51,7 +51,7 @@ This exporter supports sending OpenTelemetry logs and traces to [Elasticsearch](
   - `date_format`(default=`%Y.%m.%d`): Time format (based on strftime) to generate the second part of the Index name.
 - `pipeline` (optional): Optional [Ingest pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html) ID used for processing documents published by the exporter.
 - `flush`: Event bulk indexer buffer flush settings
-  - `bytes` (default=5242880): Write buffer flush size limit.
+  - `bytes` (default=5000000): Write buffer flush size limit.
```

Comment:
[to reviewer] There is no change in practice and the README was wrong. The number came from https://github.com/elastic/go-elasticsearch/blob/7.17/esutil/bulk_indexer.go#L204